Kernel/Scheduler.cpp

```diff
@@ -222,10 +222,22 @@ void Scheduler::pick_next()
 
     auto& thread_to_schedule = pull_next_runnable_thread();
     if constexpr (SCHEDULER_DEBUG) {
+#if ARCH(I386) || ARCH(X86_64)
         dbgln("Scheduler[{}]: Switch to {} @ {:#04x}:{:p}",
             Processor::current_id(),
             thread_to_schedule,
             thread_to_schedule.regs().cs, thread_to_schedule.regs().ip());
+#elif ARCH(AARCH64)
+        // TODO
```

Comment:
what is todo about this? it looks like it prints more or less the same information now.